scripts/build_plugins.sh

```diff
@@ -174,6 +174,6 @@ fi
 
 if [ "$OS_TYPE" == "linux" ];
 then
-  docker build $PLUGIN_BUILD_ARGS --no-cache -t aws-fluent-bit-plugins:latest -f ./Dockerfile.plugins .
+  docker build --build-arg LOCAL_BUILD=true $PLUGIN_BUILD_ARGS --no-cache -t aws-fluent-bit-plugins:latest -f ./Dockerfile.plugins .
```

Comment:
How will this work in the cloud? Have you tested it there? I think this will always set the build flag no matter what. We discussed yesterday with @ShelbyZ how to pass an env var from the Makefile to this script to set this flag.

Reply:
Still have same question

Reply:
Sorry I missed the update with BUILD_TYPE